cmd/release/docs/main.go

```diff
@@ -28,6 +29,8 @@ func main() {
 	includeREADME := *flag.Bool("includeREADME", true, "If README should be updated")
 	includeDocsIndex := *flag.Bool("includeDocsIndex", true, "If index.md in docs should be updated")
 
+	includePR := *flag.Bool("includePR", true, "If a PR should be opened for changed")
```

Comment:
NIT: I wonder if `openPR` makes more sense for this

Reply:
Done!